README.md

```diff
@@ -19,8 +19,6 @@ A wrapper around ES6 fetch to simplify interacting with JSON APIs.
 
 ```
 yarn add json-fetch
-# or...
-npm install json-fetch
```

Comment:
minor but I think it was fine to keep this - this is open source so we shouldn't make assumptions about what what others are using to manage dependencies 🤷‍♂️